source/val/validate_mode_setting.cpp

```diff
@@ -443,6 +476,18 @@ spv_result_t ValidateExecutionMode(ValidationState_t& _,
         }
       }
       break;
+    case SpvExecutionModeOutputLinesEXT:
+    case SpvExecutionModeOutputTrianglesEXT:
+    case SpvExecutionModeOutputPrimitivesEXT:
+      if (!std::all_of(models->begin(), models->end(),
+                       [](const SpvExecutionModel& model) {
+                         return model == SpvExecutionModelMeshEXT;
+                       })) {
+        return _.diag(SPV_ERROR_INVALID_DATA, inst)
+               << "Execution mode can only be used with the MeshEXT execution "
+                  "model.";
```

Comment:
This seems to break the NV version of the extension. Take a look at the shaderc smoketest.

Reply:
Ran the failed NV Mesh Shader through and should be good now, didn't realize the execution modes were aliased